evap/evaluation/templates/contact_modal.html

```diff
@@ -48,6 +48,8 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
     function {{ modal_id }}Show() {
         $('#{{ modal_id }}ActionButton').unbind().click(function(){ {{ modal_id }}Action(); });
         $('#{{ modal_id }}').modal();
+        // remove wrongly added padding by Bootstrap
```

Comment:
can you please add a comment where exactly (which file, which line) bootstraps adds the padding?